packages/app/src/specs/banners/EnableNotificationsBanner.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div
+    data-cy="enable-notifications-banner"
+    class="flex items-start flex-wrap gap-[8px] justify-between w-full px-[24px] py-[16px] bg-indigo-50 border-b border-b-indigo-100"
+  >
+    <div class="flex flex-col">
+      <div class="text-gray-900 font-medium text-base">
+        {{ t('specPage.banners.enableNotifications.title') }}
+      </div>
+      <div class="text-gray-700 text-sm mt-[3px]">
+        {{ t('specPage.banners.enableNotifications.subtitle') }}
+      </div>
+    </div>
+    <div class="flex">
+      <div>
+        <Button
+          size="40"
+          @click="enableNotifications"
+        >
+          {{ t('specPage.banners.enableNotifications.enableDesktopNotifications') }}
+        </Button>
+      </div>
+      <div class="ml-[7px]">
+        <Button
+          variant="outline-light"
+          @click="remindLater"
+        >
+          {{ t('specPage.banners.enableNotifications.remindMeLater') }}
+        </Button>
+      </div>
+      <div class="ml-[7px] text-gray-200">
+        <Button
+          aria-label="Dismiss Banner"
+          class="flex items-center h-[42px]"
+          variant="outline-light"
+          @click="dismissBanner"
+        >
+          <Icon name="action-delete-medium" />
```

Comment:
Didn't know this was a pattern. Any reason to prefer this over using the direct icon imports?

```tsx
import { IconActionDeleteMedium } from '@cypress-design/vue-icon`;

<IconActionDeleteMedium />
```